server/README.md

```diff
@@ -315,7 +315,19 @@ containing a `crt` file and point `X509_CA_BUNDLE` environmental variable to tha
 
 NOTE: See `openshift-examples` directory for an out of the box setup for OpenShift.
 
+### Enable some metrics
 
+Keycloak image can collect some statistics for various subsystem which will then be available in the management console and the `/metrics` endpoint.
+You can enable it with the following environment variables :
```

Comment:
Should be KEYCLOAK_ not WILDFLY_

EJB3 shouldn't be there as Keycloak doesn't use EJBs.

Reply:
sorry about that, I was trying to replicate what was in the server distribution. I didn't realize it was not keycloak specific, i'll fix it.